README.md

```diff
@@ -35,3 +35,14 @@ We are assuming that you're using bash & you have to install or clone such packa
         }
     }
     ```
+* Install npm and packages
+ 
+  ```
+  sudo apt-get install node-js
+  npm install -g gulp
+  npm install -g typescript
+  npm i
+  gulp
```

Comment:
Remove extra line break
```
npm i gulp
```
And perhaps will be better to use same style as above (i == install) :relieved: 

```
npm install gulp
```